packages/lesmis/src/routes/AdminPanelApp.jsx

```diff
@@ -204,38 +203,46 @@ const AdminPanelApp = ({ user }) => {
         <LogoutPage redirectTo={`${adminUrl}/login`} />
       </Route>
       <LesmisAdminRoute path={`${path}/viz-builder`} hasAdminPanelAccess={userHasAdminPanelAccess}>
-        <VizBuilderApp
-          Navbar={({ user: vizBuilderUser }) => <AdminPanelNavbar user={vizBuilderUser} />}
-        />
+        <VizBuilderApp />
       </LesmisAdminRoute>
       <PrivateRoute path={`${path}`} loginPath={`${adminUrl}/login`}>
-        <AdminPanelNavbar user={user} links={routes} />
-        <div ref={headerEl} />
-        <Switch>
-          {[...routes].map(route => (
-            <LesmisAdminRoute
-              key={route.to}
-              path={`${route.to}`}
-              hasAdminPanelAccess={userHasAdminPanelAccess}
-              render={({ match }) => {
-                return (
-                  <>
-                    <TabsToolbar links={route.tabs} maxWidth="xl" baseRoute={match.url} />
-                    <Switch>
-                      {route.tabs.map(tab => (
-                        <Route key={`${route.to}-${tab.to}`} path={`${route.to}${tab.to}`} exact>
-                          <tab.component getHeaderEl={getHeaderEl} translate={translate} />
-                        </Route>
-                      ))}
-                      <Redirect to={`${route.to}`} />
-                    </Switch>
-                  </>
-                );
-              }}
-            />
-          ))}
-          <Redirect to={`${path}/survey-responses`} />
-        </Switch>
+        <PageWrapper>
+          <AdminPanelNavbar user={user} links={routes} />
+          <Main>
+            <Switch>
+              {[...routes].map(route => (
+                <LesmisAdminRoute
+                  key={route.to}
+                  path={`${route.to}`}
```

Comment:
Looks like this and line 233 have redundant templating—can probably use the string directly(?)